service-rpc/src/main/java/org/apache/iotdb/rpc/NonOpenTransport.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.rpc;
+
+import org.apache.thrift.transport.TTransport;
+
+/**
+ * A TTransport that does not require open/close.
+ */
+public abstract class NonOpenTransport extends TTransport {
+
+  @Override
+  public void close() {
+    // do nothing
+  }
+
+  @Override
+  public boolean isOpen() { return true; }
```

Comment:
how about set to `false` once `close()` is called.

Reply:
No point. `close()` means nothing to such classes. You may find similar codes in other thrift Transports like MemTransport.